frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -417,7 +429,14 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
             'start_time' as RecordingsQuery['order'],
             { persist: true },
             {
-                setOrderBy: (_, { orderBy }) => orderBy,
+                setOrderBy: (_, { orderBy }) => orderBy as RecordingsQuery['order'],
```

Comment:
casting here to make other typing easier. it's still protected by the typing on the `setOrderBy` param